gulpfile.js

```diff
@@ -1245,9 +1246,10 @@ gulp.task('gh-pages-prepare', function () {
 gulp.task('wintersmith', function (done) {
   var wintersmith = require('wintersmith');
   var env = wintersmith('docs/config.json');
-  env.build(GH_PAGES_DIR, function (error) {
+  env.build(GH_PAGES_DIR, function(error) {
     if (error) {
-      return done(error);
+      done(error);
+      return;
```

Comment:
`return undefined`?

Reply:
Nope, because of "This rule requires `return` statements to either always or never specify values." as explained in https://eslint.org/docs/rules/consistent-return#rule-details.

Keep in mind that functions always, implicitly, returns `undefined` unless something else is specified.

Reply:
D'oh, I misread how the rule works. Thanks.